docker/testnet/Dockerfile

```diff
@@ -9,8 +9,10 @@ RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
 
 USER cheqd
 
-COPY docker/single_image_testnet/gen_node_configs.sh .
-COPY docker/single_image_testnet/supervisord.conf /etc
+COPY --chown=cheqd:cheqd docker/testnet/gen_node_configs.sh ./
+COPY docker/testnet/supervisord.conf /etc
+
+RUN chmod +x ./gen_node_configs.sh
 
 EXPOSE 26657 26659
```

Comment:
Currently, the "single image testnet" folder is for a local network. This docker/testnet folder should be for the production testnet, so for instance these ports are built under the assumption that it's for local network when actually for production testnet this should be very, very similar to mainnet.

Reply:
Done, moved to single_image_testnet, left testnet intact as a docker-compose only, meant for production.